src/functionalTest/java/uk/gov/hmcts/reform/locationrefdata/cameltest/LrdApplicationExceptionAndAuditTest.java

```diff
@@ -43,8 +43,9 @@
 @CamelSpringBootTest
 @MockEndpoints("log:*")
 @ContextConfiguration(classes = {LrdCamelConfig.class, CamelTestContextBootstrapper.class,
-    JobLauncherTestUtils.class, BatchConfig.class, AzureBlobConfig.class, BlobStorageCredentials.class},
+    BatchConfig.class, AzureBlobConfig.class, BlobStorageCredentials.class},
     initializers = ConfigDataApplicationContextInitializer.class)
+@SpringBatchTest
```

Comment:
`JobLauncherTestUtils` is no longer initialised automatically, but `@SpringBatchTest` creates it.